config/src/config/consensus_config.rs

```diff
@@ -179,42 +179,57 @@ impl Default for ConsensusConfig {
 
             // Voting backpressure is only used as a backup, to make sure pending rounds don't
             // increase uncontrollably, and we know when to go to state sync.
-            vote_back_pressure_limit: 30,
+            // Considering block gas limit and pipeline backpressure should keep number of blocks
+            // in the pipline very low, we can keep this limit pretty low, too.
+            vote_back_pressure_limit: 12,
             pipeline_backpressure: vec![
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 1000,
+                    back_pressure_pipeline_latency_limit_ms: 800,
                     max_sending_block_txns_override: 10000,
                     max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backpressure_proposal_delay_ms: 100,
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 1500,
+                    back_pressure_pipeline_latency_limit_ms: 1100,
```

Comment:
Is 300 ms enough to avoid the backpressure being "stuck" at some value? It's "exactly" enough right (100ms * 3 difference from the previous value)? Should some buffer be added?

Reply:
what is 3*difference? 

we don't have two roundtrips counted here any more, so 100 delay should only ever add 100 to values, so it has 200 ms to work with